### PR TITLE
Add doc for multi value GEO

### PR DESCRIPTION
**Pull request by oshadmi**

### CODE REVIEW

File: docs/docs/Indexing_JSON.md [hunk 1]
```diff
@@ -280,6 +280,87 @@ When JSONPath leads to multiple numerical values:
...
 
+## Index JSON arrays as GEO
+
+Starting with RediSearch 2.6.1, search can be done on an array of geo (geographical) values or on a JSONPath leading to multiple geo values.
+
+Prior to RediSearch 2.6.1, only a single geo value was supported per GEO attribute. The geo value was specified using a comma delimited string in the form "longtitude,latitude", for example, "15.447083,78.238306".
+
+With RediSearch 2.6.1, a JSON array of such geo values is also supported.
+
+In order to index multiple geo values, either use a JSONPath leading to a single array of geo values, or a JSONPath leading to multiple geo values, using JSONPath operators such as wildcard, filter, union, array slice, and/or recursive descent.
```
Comment (nermiller) [inline docs/docs/Indexing_JSON.md:291]:
```suggestion
In order to index multiple geo values, use either  a JSONPath leading to a single array of geo values, or a JSONPath leading to multiple geo values, using JSONPath operators such as wildcard, filter, union, array slice, and/or recursive descent.
```

File: docs/docs/Indexing_JSON.md [hunk 2]
```diff
@@ -280,6 +280,87 @@ When JSONPath leads to multiple numerical values:
...
+
+To index the `location` geo array, specify the JSONPath `$.location` in the `SCHEMA` definition during index creation:
+
+
+```sql
+127.0.0.1:6379> FT.CREATE vendorIdx ON JSON PREFIX 1 vendor: SCHEMA $.location AS loc GEO
+OK
+```
+
+Now we can search for a vendor which is close by to a specific location. For example, a customer is located at geo coordinates 34.5,31.5 and we want to get the vendors which are within the range of 40 km from our location:
```
Comment (nermiller) [inline docs/docs/Indexing_JSON.md:338]:
```suggestion
Now we can search for a vendor close to a specific location. For example, a customer is located at geo coordinates 34.5,31.5 and we want to get the vendors that are within the range of 40 km from our location:
```

**Review by nermiller — DISMISSED** [review]
Just a couple of nits, @oshadmi

File: docs/docs/Indexing_JSON.md [hunk 3]
```diff
@@ -280,6 +280,87 @@ When JSONPath leads to multiple numerical values:
   - `null` values are skipped
   - Any other value type is causing an indexing failure
 
+## Index JSON arrays as GEO
+
+Starting with RediSearch 2.6.1, search can be done on an array of geo (geographical) values or on a JSONPath leading to multiple geo values.
```
Comment (ashtul) [inline docs/docs/Indexing_JSON.md:285]: Should we use v2.6?

**Review by ashtul — COMMENTED** [review]
Looks good.

File: docs/docs/Indexing_JSON.md (hunk 3 above)
Comment (oshadmi) [reply docs/docs/Indexing_JSON.md:285]: Some capabilities are available already with 2.6.0 (multi value TEXT)
Some are available with 2.6.1 (multi value NUMERIC and GEO)

**Review by rafie — APPROVED** [review]

**Pull request merged**